privval/remote_signer.go

```diff
@@ -152,7 +154,8 @@ type RemoteSignerMsg interface{}
 
 func RegisterRemoteSignerMsg(cdc *amino.Codec) {
 	cdc.RegisterInterface((*RemoteSignerMsg)(nil), nil)
-	cdc.RegisterConcrete(&PubKeyMsg{}, "tendermint/remotesigner/PubKeyMsg", nil)
+	cdc.RegisterConcrete(&PubKeyRequest{}, "tendermint/remotesigner/PubKeyRequest", nil)
+	cdc.RegisterConcrete(&PubKeyResponse{}, "tendermint/remotesigner/PubKeyResponse", nil)
```

Comment:
Splitting up `PubKeyMsg` into  `PubKeyRequest ` and `PubKeyResponse` is the reason this is marked as breaking. If that blocks merging this PR I can revert this change and follow up in a separate PR.

Reply:
this is fine. we're prepping a breaking release now